duckduckgo/BrowserServicesKit
# Add API for stored failed compilations

**Required**:

Task/Issue URL: https://app.asana.com/0/1203790657351911/1205404596674276/f
iOS PR: https://github.com/duckduckgo/iOS/pull/2014/
macOS PR: 
What kind of version bump will this require?: Minor

**Description**:

Introduce daily pixel containing information about failed content blocking rules compilations.

**Steps to test this PR**:
1. Run the app and make sure the pixel is not being sent.
1. Make compilation fail, by for instance adding some incorrect characters to temporary unprotected sites.
1. Notice the counter increases for a given step of compilation.
1. See that the pixel is being sent on the next app run and that statistics are being reset. 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
